commit 00625ada0860e3c39dd79bc13fb176a7f40e890a (refs/changes/35/3235/1)
Author: Josh Wilsdon <josh@wilsdon.ca>
Date:   2018-01-18T14:22:34-08:00 (1 year, 9 months ago)
    
    TRITON-64 index filenames should be index.platform.js instead of platform.index.js

diff --git a/lib/index.js b/lib/index.js
index cb09e81..5b503f7 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -9,4 +9,4 @@
  */
 
 var os = require('os');
-module.exports = require('./' + os.platform() + '.index');
+module.exports = require('./index.' + os.platform());
diff --git a/lib/sunos.index.js b/lib/index.sunos.js
similarity index 100%
rename from lib/sunos.index.js
rename to lib/index.sunos.js
